### PR TITLE
use sdk to run tests

### PR DESCRIPTION
**Pull request by Ankit098**

### CODE REVIEW

File: tests/local-test.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import json
+from selenium import webdriver
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options as ChromeOptions
+
+options = ChromeOptions()
+driver = webdriver.Remote(
+    command_executor='http://localhost:4444/wd/hub',
```
Comment (kamal-kaur04) [inline tests/local-test.py:14]: Don't think sample should ideally point to local port.

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

File: tests/local-test.py (hunk 1 above)
Comment (Ankit098) [reply tests/local-test.py:14]: the idea is that if this is a test user is running locally they don't need to change it just need to add a yml and use browserstack sdk

Comment (Ankit098) [reply tests/local-test.py:14]: @alliv8 thoughts?

Comment (alliv8) [reply tests/local-test.py:14]: Yea it's good to show how our demo works and that the SDK works with any web driver

Comment (alliv8) [reply tests/local-test.py:14]: Let's add a code comment that this will be overridden when invoking the sdk

**Review by kamal-kaur04 — APPROVED** [review]

**Pull request merged**